internal/http/types.go

```diff
@@ -87,9 +87,37 @@ type FluxNotifyRequest struct {
 	FluxEvent   event.Event
 }
 
+type DeploymentEvent struct {
+	Name          string `json:"name,omitempty"`
+	Namespace     string `json:"namespace,omitempty"`
+	AvailablePods int32  `json:"podCount,omitempty"`
+	Replicas      int32  `json:"replicas,omitempty"`
+	ArtifactID    string `json:"artifactId,omitempty"`
+	AuthorEmail   string `json:"authorEmail,omitempty"`
+	Environment   string `json:"environment,omitempty"`
+}
+type ContainerError struct {
+	Name         string `json:"name"`
+	ErrorMessage string `json:"message"`
+	Type         string `json:"type"`
```

Comment:
We could omit empty values here.

```suggestion
	Name         string `json:"name,omitempty"`
	ErrorMessage string `json:"message,omitempty"`
	Type         string `json:"type,omitempty"`
```